src/main/java/org/apache/commons/crypto/NativeCodeLoader.java

```diff
@@ -171,7 +171,7 @@ private static File extractLibraryFile(final String libFolderForCurrentOS, final
             // Check whether the contents are properly copied from the resource
             // folder
             try (InputStream nativeInputStream = NativeCodeLoader.class.getResourceAsStream(nativeLibraryFilePath)) {
-                try (InputStream extractedLibIn = new FileInputStream(extractedLibFile)) {
+                try (InputStream extractedLibIn = Files.newInputStream(Paths.get(extractedLibFile.getPath()))) {
```

Comment:
See line 148 above, already a path I think.

Reply:
well seen. changed

Reply:
Do you habe tested this for speed? I saw (and I don’t know why) performance regressions with the stream of Files.newInputStream compared to FileInputStream. Maybe that should be a separate change to introduce Path/NIO?

Reply:
> Do you habe tested this for speed? I saw (and I don’t know why) performance regressions with the stream of Files.newInputStream compared to FileInputStream. Maybe that should be a separate change to introduce Path/NIO?

Hi,
There is High Priority warning that recomend avoid FileStream. Cause garbage collection pauses

Reply:
What warning? Issued by whom?

Reply:
https://pmd.github.io/latest/pmd_rules_java_performance.html#avoidfilestream
https://bugs.openjdk.java.net/browse/JDK-8080225

Reply:
@arturobernalg 
Thanks for the pointers.